docs/upgrade/11.11_to_11.12/mariadb.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX IF NOT EXISTS IX_oc_aws_asset_mapping_object_key ON oc_aws_asset_mapping (object_key);
```

Comment:
Hi rute, thanks for your PR, now that is aiming to develop, can you change the folder name to `12_to_13` for both scripts. Thanks

Reply:
Oops! Fixing...